lib/ajax-datatables-rails/orm/active_record.rb

```diff
@@ -38,31 +38,28 @@ def composite_search(records)
         records
       end
 
-      def build_conditions_for(query)
+      def build_conditions_for(query, regex)
         search_for = query.split(' ')
         criteria = search_for.inject([]) do |criteria, atom|
-          criteria << searchable_columns.map { |col| search_condition(col, atom) }
+          criteria << searchable_columns.map { |col| search_condition(col, atom, regex == 'true') }
             .reduce(:or)
         end.reduce(:and)
         criteria
       end
 
       def aggregate_query
-        conditions = searchable_columns.each_with_index.map do |column, index|
+        conditions = view_columns.each_with_index.map do |column, index|
```

Comment:
haha :smiley:  no worries! I think though option #2 from above:
Perhaps better, we iterate over searchable_columns, but also filter out the columns from params[:columns] that are searchable and check for their value at that index.

would be a good thing for me to put together. That way, we can iterate over an array of the `params[:columns]` that only contains the ones that are searchable, thus saving a couple useless iterations. Plus having a method that grabs the params of the searchable columns (and another for sortable) would probably be handy to have anyways.